kgraphql/src/test/kotlin/com/apurebase/kgraphql/schema/dsl/DataLoaderPropertyDSLTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.apurebase.kgraphql.schema.dsl
+
+import com.apurebase.kgraphql.defaultSchema
+import org.junit.Test
```

Comment:
Small issue here, we are currently using `import org.junit.jupiter.api.Test`.

Reply:
Done